src/routes/api/collabs/[id]/picks/[pick_id]/delimage.ts

```diff
@@ -0,0 +1,110 @@
+import type { RequestHandler } from '@sveltejs/kit';
+
+import cookie from 'cookie';
+import { type Pick } from '@prisma/client';
+import { Jwt } from '../../../../../../jwt';
+import { getUser } from '../../../../discord/user';
+import { Prisma } from '../../../../../../database/prisma';
+import { Env } from '../../../../../../env';
+import { SentryClient } from '../../../../../../bot/sentry';
+import { existsSync, unlinkSync } from 'fs';
+import path from 'path';
+
+export async function deleteImage(pick: Pick): Promise<void> {
+	if (!pick) {
+		throw new Error('Pick not found');
+	}
+
+	const env = Env.load();
+
+	if (pick.image) {
+		const filePath = path.join(
+			env['FILE_STORAGE_PATH'],
+			'collabs',
+			pick.collabId,
+			'picks',
+			pick.image
+		);
+
+		if (existsSync(filePath)) {
+			unlinkSync(filePath);
+		}
+	}
+
+	await Prisma.client.pick.update({
+		where: {
+			id: pick.id
+		},
+		data: { image: null }
+	});
+}
+
+export const del: RequestHandler = async ({ request, params }) => {
+	const cookieHeader = request.headers.get('cookie');
+	const cookies = cookie.parse(cookieHeader ?? '');
+	const decoded = Jwt.decode(cookies['discord_token']);
+	const token = decoded['access_token'] as string;
+
+	const decodedUser = Jwt.decode(cookies['user_id']);
+	const userId = decodedUser['user_id'] as string;
+
+	if (!token) {
+		return {
+			status: 401
+		};
+	}
+
+	try {
+		const user = await getUser(token, userId);
+
+		if (!user) {
```

Comment:
Do the admin check here to save some database calls

Reply:
cant get out of the user call but put it after to avoid calling the pick